### PR TITLE
04. Describe the main page filter

### LINKED ISSUE #4: 04. Describe the main page filter

### PR DESCRIPTION
**Pull request by maksym-bielyshev**

### CODE REVIEW

File: pages/home_page.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
+from selenium.webdriver import Remote
+from pages.base_page import BasePage
+from locators import LocatorsHomeFilter
+from components import DropdownComponent
+
+
+class HomePage(BasePage):
+    def __init__(self, driver: Remote):
```
Comment (denyskulykov) [inline pages/home_page.py:10]: Add class sting

File: locators.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        By.XPATH,
+        "//body/div[@id='app']/main[1]/div[2]/div[2]/form[1]/div[1]/label[2]"
+    )
+    NEW_AUTOS = (
+        By.XPATH,
+        "//body/div[@id='app']/main[1]/div[2]/div[2]/form[1]/div[1]/label[3]"
+    )
+    AUTOS_FOR_DELIVERY = (
+        By.XPATH,
+        "//body/div[@id='app']/main[1]/div[2]/div[2]/form[1]/div[1]/label[4]"
```
Comment (denyskulykov) [inline locators.py]: Change localors

File: pages/home_page.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+            *LocatorsHomeFilter.PRICE_FROM
+        )
+        self.price_to = driver.find_element(
+            *LocatorsHomeFilter.PRICE_TO
+        )
+        self.search_button = driver.find_element(
+            *LocatorsHomeFilter.SEARCH_BUTTON
+        )
+
+    def click_all_autos(self):
```
Comment (denyskulykov) [inline pages/home_page.py:42]: Add annotations

File: locators.py (hunk 2 above)
Comment (maksym-bielyshev) [reply locators.py]: Locators are changed.

File: pages/home_page.py (hunk 3 above)
Comment (maksym-bielyshev) [reply pages/home_page.py:42]: Annotations (except for the 'self') are added.

File: pages/home_page.py (hunk 1 above)
Comment (maksym-bielyshev) [reply pages/home_page.py:10]: Class string is added

**Review by denyskulykov — APPROVED** [review]

**Pull request merged**